scripts/communityScripts/chat/FloofChat.js

```diff
@@ -80,7 +85,7 @@ function init() {
     setupHistoryWindow(false);
 
     chatBar = new OverlayWindow({
-        source: Paths.defaultScripts + '/communityModules/chat/FloofChat.qml?' + Date.now(),
+        source: Paths.defaultScripts + '/communityScripts/chat/FloofChat.qml?' + Date.now(),
```

Comment:
It would be better to use Scripts.resolvePath() to get the QML relative to the script file rather than "hard-coded" Paths.defaultScripts value which evaluates to C:/Program Files/...etc. per install or build directory. (The latter makes it harder to test app changes during development.)

Reply:
Better code...
```
source: ROOT + "FloofChat.qml?" + Date.now(),
```